runner/request.go

```diff
@@ -5,15 +5,28 @@ import (
 	"crypto/tls"
 	"io/ioutil"
 	"net/http"
+	"net/url"
+	"os"
 	"strings"
 
 	"github.com/lamoda/gonkey/models"
 )
 
-func newClient() *http.Client {
-	return &http.Client{Transport: &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-	}}
+func newClient() (*http.Client, error) {
+	if os.Getenv("HTTP_PROXY") != "" {
+		proxyUrl, err := url.Parse(os.Getenv("HTTP_PROXY"))
+		if err != nil {
+			return nil, err
+		}
+		return &http.Client{Transport: &http.Transport{
```

Comment:
It could be written more clear:
```
		return &http.Client{Transport: &http.Transport{
			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
                         Proxy: ?
		}}, nil
```
is the common part of both branches, so it can be moved out of `if`.

Reply:
done